frontend/src/components/admin-page/admin-page.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { useDispatch } from 'react-redux';
+import { UsersActionCreator } from 'store/slices';
+import styles from './styles.module.scss';
+import AdminTable from './admin-table';
+import CreateUser from './create-user';
+import EditUser from './edit-user';
+import { IEditUserPayload, IRegisterPayload, IUser } from 'common/interfaces';
+import { DEFAULT_USER_INSTANCE } from 'common/constants';
+
+const AdminPage: React.FC = () => {
+  const [user, setUser] = React.useState(DEFAULT_USER_INSTANCE);
+  const [showPopUp, setShowPopUp] = React.useState(false);
+
+  const dispatch = useDispatch();
+  const editUser = (data: IEditUserPayload) => {
```

Comment:
```suggestion
  const handleEditUser = (data: IEditUserPayload) => {
```
Use `handle<Element><Event>` for local component's cbs
User `on<Element><Event>` for props component's cbs